pkg/dispatchcli/cmd/update_secret.go

```diff
@@ -35,9 +36,12 @@ func CallUpdateSecret(input interface{}) error {
 	client := secretStoreClient()
 	secretBody := input.(*models.Secret)
 
-	params := secret.NewUpdateSecretParams()
-	params.Secret = secretBody
-	params.SecretName = *secretBody.Name
+	params := &secret.UpdateSecretParams{
```

Comment:
I'm pretty sure this will break. If you don't use the NewUpdateSecretParams() function then you need to set the default timeout yourself otherwise the update will fail with a very meaningless error. At least I noticed that when adding the update for base images.

To set the timeout you just need to add `timeout: cr.DefaultTimeout,` where `cr` is `cr "github.com/go-openapi/runtime/client"`

Reply:
Thanks for the info! I will do that